decidim-core/db/migrate/20170221094835_add_scopes_to_processes.rb

```diff
@@ -0,0 +1,5 @@
+class AddScopesToProcesses < ActiveRecord::Migration[5.0]
+  def change
+    add_column :decidim_participatory_processes, :scope_ids, :integer, array: true, default: []
```

Comment:
@oriolgual didn't we agree to support multiple scopes for a process in the backend, but in fact only relate to one by now? I was sure we agreed to this, but I might have dreamt it :/